fourthExample.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState, useEffect} from 'react';
+
+const FourthExample = () => {
+    const [data, setData] = useState()
+
+    const fetchData = async () => {
+        return await fetch("url");
+    }
+
+    //useEffect expect return value from first argument will be undefined
```

Comment:
Exactly, the function provided to `useEffect` cannot return `Promise` (like all `async` functions do).